File: 117/11799/HorrorDash.java
import java.util.Scanner;

class Main {
    public static void main(String[] args) {
        Scanner scan = new Scanner(System.in);
        int tests = scan.nextInt();
        for (int t = 1; t <= tests; t++) {
            int n = scan.nextInt();
            int maxSpeed = 0;
            for (int i = 0; i < n; i++) {
                int speed = scan.nextInt();
                if (speed > maxSpeed)
                    maxSpeed = speed;
            }
            System.out.format("Case %d: %d\n", t, maxSpeed);
        }
    }
}

File: uva/11799/horrordash.c
#include <stdio.h>

int main() {
    int tests, t;
    scanf("%d", &tests);
    for (t = 1; t <= tests; t++) {
        int n, i;
        int maxSpeed = 0;
        scanf("%d", &n);
        for (i = 0; i < n; i++) {
            int speed;
            scanf("%d", &speed);
            if (speed > maxSpeed)
                maxSpeed = speed;
        }
        printf("Case %d: %d\n", t, maxSpeed);
    }
    return 0;
}
